### PR TITLE
Add LyricsFinder.app v1.5.6

### PR DESCRIPTION
**Pull request by jukremer**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/l/lyricsfinder.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  livecheck do
+    url :homepage
+    regex(/"softwareVersion">(\d+(?:\.\d+)+)</i)
+  end
+
+  depends_on macos: ">= :el_capitan"
+
+  app "LyricsFinder.app"
+
+  zap trash: ""
```
Comment (SMillerDev) [inline Casks/l/lyricsfinder.rb:19]: That doesn't seem correct

Comment (janurskremer) [reply Casks/l/lyricsfinder.rb:19]: Can I edit the code or do I have to make a new PR? 
It should be:
`zap trash: "~/Library/Preferences/com.mediahuman.Lyrics Finder.plist"`

Comment (SMillerDev) [reply Casks/l/lyricsfinder.rb:19]: Please don't make a new PR, editing it is the preferred way.

Comment (janurskremer) [reply Casks/l/lyricsfinder.rb]: Ok should be fixed now.

File: Casks/l/lyricsfinder.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+cask "lyricsfinder" do
+  version "1.5.6"
+  sha256 :no_check
+
+  url "https://www.mediahuman.com/download/LyricsFinder.dmg"
+  name "LyricsFinder"
+  desc "Free software application that can help you find and add missing lyrics"
```
Comment (bevanjkay) [inline Casks/l/lyricsfinder.rb:7]:
```suggestion
  desc "Find and download song lyrics"
```

File: Casks/l/lyricsfinder.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+cask "lyricsfinder" do
+  version "1.5.6"
+  sha256 :no_check
+
+  url "https://www.mediahuman.com/download/LyricsFinder.dmg"
+  name "LyricsFinder"
```
Comment (bevanjkay) [inline Casks/l/lyricsfinder.rb:6]:
```suggestion
  name "Lyrics Finder"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @janurskremer

**Pull request merged**